sources/packages/backend/apps/queue-consumers/src/processors/schedulers/cas-integration/models/cas-supplier-response.dto.ts

```diff
@@ -0,0 +1,52 @@
+export class CASSupplierResponse {
+  items: CASSupplierResponseItem[];
+  hasMore: boolean;
+  limit: number;
+  offset: number;
+  count: number;
+  links: CASSupplierResponseItemLink[];
+}
+export class CASSupplierResponseItem {
+  suppliernumber: string;
+  suppliername: string;
+  subcategory: string;
+  sin: string;
+  providerid?: string;
+  businessnumber?: null;
+  status: string;
+  supplierprotected?: string | null;
+  standardindustryclassification?: string;
+  lastupdated: string;
+  supplieraddress: CASSupplierResponseItemAddress[];
+}
+
+export class CASSupplierResponseItemAddress {
+  suppliersitecode: string;
+  addressline1: string;
+  addressline2?: string;
+  addressline3?: string;
```

Comment:
Maybe the business missed the `addressline2` and `addressline3` in the requirement. Have we ever asked then?
If not we should ask, if not in this PR, we can be added to an upcoming ticket.